unit-tests/live/options/test-rgb-options-metadata-consistency.py

```diff
@@ -7,6 +7,10 @@
 import pyrealsense2 as rs
 from rspy import test
 
+def close_resources(sensor):
+    sensor.stop()
```

Comment:
What is it wasn't started/opened? Need checking?

Reply:
👍